tidy3d/components/simulation.py

```diff
@@ -6,8 +6,12 @@
 import numpy as np
 import xarray as xr
 import matplotlib.pylab as plt
-import matplotlib as mpl
-from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+try:
+    import matplotlib as mpl
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+except Exception:  # pylint: disable=broad-except
```

Comment:
hm, this is probably not great practice.  do you think eventually we can find a solution?

Reply:
Yes, eventually...